### ISSUE TITLE
update AMY URL

### ISSUE THREAD

**Issue by fmichonneau** [carpentries/carpentries.org-archive#383]
We need to replace AMY's URL everywhere to use its new carpentries.org domain.

**Comment by maneesha** [comment]
Hi @fmichonneau I am trying this by doing 
`grep -rl  'amy.software-carpentry.org' | xargs sed -i 's,amy.software-carpentry.org,amy.carpentries.org,g'`  
but get the error
 `xargs: unmatched single quote; by default quotes are special to xargs unless you use the -0 option`

Can you help me understand why this is happening?

**Comment by maneesha** [comment]
Looks like it just didn't like something in the `_site` directory.  Not sure why, but this worked:
```
grep -rl --exclude-dir=_site 'amy.software-carpentry.org' | xargs sed -i 's,amy.software-carpentry.org,amy.carpentries.org,g'
```

Fixed with 73b95d4
